scripts/create-dmg.sh

```diff
@@ -14,7 +14,7 @@ cmake \
     -DCMAKE_BUILD_TYPE=Release \
     "$@" \
     ../.. || exit 1
-make -j || exit 1
```

Comment:
If this ends up fixing the problem, I think the best solution would be to set the parallelism to the number of cores the machine has. `make -j` can spawn an excessive number of threads -- more than there are cores available -- sometimes leading to OOM.

The way I'd do this is to change the build script to entirely ditch makefiles and use ninja instead :p. I.e. add -GNinja to the configure command and replace the `make` call with `cmake --build . --config Release -j`

Reply:
I was seeing `clang: error: unable to execute command: posix_spawn failed: Resource temporarily unavailable` pop up sporadically, so I figured I'd try [this StackOverflow answer](https://stackoverflow.com/a/46255347).

My secondary guess is that Github's ARM Runners are a hot commodity, and each one is stressed out sharing a bunch of jobs 😅 

It looks like it did work!   And it built like 4 times _more quickly_ 🎉: https://github.com/zalo/tev/actions/runs/14750446001/job/41406464717

Reply:
🐌 Race started between:
```machine_type=$(uname -m)
if [[ "$machine_type" == "arm"* ]]; then
  make || exit 1
else
  make -j || exit 1
fi
```
https://github.com/zalo/tev/actions/runs/14750886621

And
```
cmake \
    -GNinja \
    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.15 \
    -DCMAKE_BUILD_TYPE=Release \
    "$@" \
    ../.. || exit 1
cmake --build . --config Release -j || exit 1
```
https://github.com/zalo/tev/actions/runs/14751011995

Winner gets it 😄

Reply:
Conditional `make -j`:
![image](https://github.com/user-attachments/assets/b8e8434c-a4b7-4f6d-8e66-bc600d3d58a9)

Ninja:
![image](https://github.com/user-attachments/assets/80c01b68-7ae4-4ae4-8553-dba96e6b8006)

Seems like it _does_ make ARM a lot faster...

Reply:
There's probably also a lot of measurement noise involved, but imho the ninja solution is the cleanest either way. I've yet to run into a cmake-based project where ninja isn't just an all-around better build system than make.

Reply:
Alrighty; I'll mark this conversation as resolved then 👍